evap/staff/templates/staff_email_preview_form.html

```diff
@@ -1,24 +1,24 @@
 <div class="card mb-3">
     <div class="card-body">
-         {% if heading %}<h4 class="card-title">{{ heading }}</h4>{% endif %} 
+         {% if heading %}<h4 class="card-title">{{ heading }}</h4>{% endif %}
         <div class="form-check mb-2">
             <input class="form-check-input" id="send_email{{ id_suffix }}" type="checkbox" name="send_email{{ id_suffix }}"
                 checked onclick="$('#email_form{{ id_suffix }}').toggle();" />
             <label class="form-check-label" for="send_email{{ id_suffix }}">{% trans 'Send email notifications' %}</label>
         </div>
         <div class="email-form" id="email_form{{ id_suffix}}">
             <div class="form-group d-flex">
-                <div class="form-label">Email subject</div>
+                <div class="form-label">{% trans 'Email subject' %}<div>
```

Comment:
`</div>` :)

Reply:
Aaah I was wondering why the site is so weird an didnt find the issue, thanks :D